src/xdocs/property_types.xml

```diff
@@ -820,6 +812,14 @@
       </div>
     </section>
 
+    <section name="RegularExpression">
+      <p>
+        This type represents a regular expression. The string representation is parsed using
+        <a href="https://docs.oracle.com/javase/7/docs/api/index.html?java/util/regex/package-summary.html">
```

Comment:
May be link should be updated as well to point to docs for release 8

Reply:
please update, as other changes are required in this PR.

Reply:
Done